src/tests/routes/test_alerts.py

```diff
@@ -31,6 +31,10 @@
     {"id": 2, "login": "connected_user", "access_id": 2, "created_at": "2020-11-13T08:18:45.447773"},
 ]
 
+GROUP_TABLE = [
+    {"id": 1, "name": "test_group"},
+]
+
```

Comment:
Let's remove everything in the unittests that doesn't involve groups yet (so everything apart from test_groups.py)